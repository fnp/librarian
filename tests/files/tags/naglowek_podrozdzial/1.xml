<naglowek_podrozdzial> numer-i/lub-tytuł-podrozdziału </naglowek_podrozdzial>
