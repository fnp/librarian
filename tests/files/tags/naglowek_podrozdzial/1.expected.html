<div id="book-text"><h4 id="s1"> numer-i/lub-tytuł-podrozdziału </h4></div>
